package-lock.json

```diff
@@ -1,8 +1,5446 @@
 {
   "name": "parsedocs",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Please revert clock file changes, unless you made some intentional changes?

Reply:
I havent touch that file, I think was auto-generated when I run npm install or something... 🤔, I changed it to 1